### DISCUSSION TITLE
Script exception: Bad item value:

### DISCUSSION THREAD

**Discussion by Drok00** [loathers/garbage-collector#1836] [Q&A] [answered]
I'm getting an odd error that kills the script running. the red line is "Script exception: Bad item value: " any ideas on troubleshooting? I do have session logs available.

**Comment by gausie** [comment]
Check the other issues before posting

**Reply by Drok00** [reply]
sorry, will close the discussion.

**Comment by Drok00** [comment, marked as answer]
Duplicated
